packages/tooltips/src/styled/StyledTooltip.ts

```diff
@@ -121,30 +121,49 @@ const sizeStyles = ({
 };
 
 const colorStyles = ({ theme, type }: IStyledTooltipProps & ThemeProps<DefaultTheme>) => {
-  let border;
   let boxShadow = theme.shadows.lg(
     `${theme.space.base}px`,
     `${theme.space.base * 2}px`,
-    getColorV8('chromeHue', 600, theme, 0.15)!
+    getColor({
+      theme,
+      hue: 'neutralHue',
+      shade: 1200,
+      light: { transparency: theme.opacity[200] },
+      dark: { transparency: theme.opacity[1100] }
+    })
   );
-  let backgroundColor = getColorV8('chromeHue', 700, theme);
-  let color = getColorV8('background', 600 /* default shade */, theme);
+
+  let backgroundColor = getColor({
+    theme,
+    hue: 'neutralHue',
+    light: { shade: 900 },
+    dark: { shade: 700 }
+  });
+
+  let borderColor = backgroundColor;
+  let color = theme.palette.white;
```

Comment:
```suggestion
  let color = getColor({ hue: 'white' });
```

Ensure a best possible future-proof lookup.